pkg/occlient/occlient.go

```diff
@@ -95,7 +95,7 @@ const (
 
 	// Default Image that will be used containing the supervisord binary and assembly scripts
 	// use getBoostrapperImage() function instead of this variable
-	defaultBootstrapperImage = "quay.io/openshiftdo/init:0.10.0"
+	defaultBootstrapperImage = "quay.io/mohammedzee1000/odo-init-image:go-init"
```

Comment:
Just so we don' forget. This needs to point to  `quay.io/openshiftdo/init:0.11.0` once the https://github.com/openshift/odo-init-image/pull/35 and new version of the image is released.

Reply:
Yep, that is why i specifically marked that change in a separate commit, so that it can be dropped :)

Reply:
`quay.io/openshiftdo/init:0.11.0` is built. @mohammedzee1000 please update it to this image

Reply:
updated